modules/ROOT/pages/create-waf-policy.adoc

```diff
@@ -25,12 +25,14 @@ When you hover over the `i` to the right of each rule, the rule ID range for tha
 image::waf-rule-set.png[]
 +
 . In *Advanced performance options*, select from the following options: +
```

Comment:
You can only choose one option, so I'd say "select one of the following options" here and in the same section below.